catalog/designers/paulhanslow/bio.html

```diff
@@ -0,0 +1 @@
+<p>Paul Hanslow is an Australian born typeface designer, currently working for Tiro Typeworks in Vancouver, Canada.</p>
```

Comment:
I suggest `Australian-born` with a hyphen to be clearer

Reply:
> I suggest `Australian-born` with a hyphen to be clearer

I would leave it as it is. The text was provided for John Hudson who pays a lot of attention to details.